packages/components/src/presentation-components/tree/controlled/PresentationTreeRenderer.tsx

```diff
@@ -99,6 +99,7 @@ function TreeNodeFilterBuilderDialog(props: TreeNodeFilterBuilderDialogProps) {
       imodel={imodel}
       descriptor={filteringInfo.descriptor}
       initialFilter={filteringInfo.active}
+      filterNodeKeys={[filterNode.key]}
```

Comment:
Might want to wrap `[filterNode.key]` into `useMemo` to avoid creating new array on each render. Keys are later used as `useEffect` dependency in `UniquePropertyValueSelector` and will cause that `useEffect` to run on each render.